lib/dish.rb

```diff
@@ -0,0 +1,28 @@
+class Dish
+
+  def initialize(name, price)
+    @name = name
+    @price = price
+  end
+
+  def read_name
+    @name.dup
```

Comment:
I know you will have a good reason for using dup - if you have time would you be able to respond with your rationale? I'd love to learn why it's better than returning the original instance variable.